src/NuGet.Services.Entities/Package.cs

```diff
@@ -285,5 +285,7 @@ public bool HasReadMe
         /// A flag that indicates that the package metadata had an embedded icon specified.
         /// </summary>
         public bool HasEmbeddedIcon { get; set; }
+
+        public EmbeddedReadmeFileType EmbeddedReadmeFileType { get; set; }
```

Comment:
EmbeddedLicenseType for Licenses?
Maybe we also use "EmbeddedReadmeType"? It's not a problem at all. I just highlight this here, and maybe others will have some opinions here.

Reply:
I also slightly prefer `EmbeddedReadmeType`, but I don't feel strongly about it either

Reply:
Yes, EmbeddedLicenseType for Licenses. I will change to 

EmbeddedReadmeType
```suggestion
        public EmbeddedReadmeType EmbeddedReadmeType { get; set; }
```

 to be consistent